mmcv/utils/parrots_wrapper.py

```diff
@@ -4,12 +4,22 @@
 
 TORCH_VERSION = torch.__version__
 
+is_rocm_pytorch = False
+if torch.__version__ >= '1.5':
```

Comment:
The version of torch can be obtained by `from mmcv.utils import TORCH_VERSION` and we should use LooseVersion to check the version. refer to #1158

```python
from distutils.version import LooseVersion
from mmcv.utils import TORCH_VERSION

if LooseVersion(TORCH_VERSION) >= LooseVersion('1.5'):
  ...
```